src/dbos.ts

```diff
@@ -1259,140 +1285,21 @@ export class DBOS {
    */
   static startWorkflow<T extends object>(targetClass: T, params?: StartWorkflowParams): InvokeFunctionsAsync<T>;
   static startWorkflow<T extends object>(target: T, params?: StartWorkflowParams): InvokeFunctionsAsync<T> {
-    if (typeof target === 'function') {
-      return DBOS.#proxyInvokeWF(target, null, params) as unknown as InvokeFunctionsAsync<T>;
-    } else {
-      return DBOS.#proxyInvokeWF(target, target as ConfiguredInstance, params) as unknown as InvokeFunctionsAsync<T>;
+    const instance = typeof target === 'function' ? null : (target as ConfiguredInstance);
```

Comment:
Are we going to deprecate `startWorkflow` in favor of `startWorkflowFunction`? I would like to have a single function for starting a workflow because it's easier to use.

Reply:
Yes, but not in this PR.